libraries/Microsoft.Bot.Builder/AutoSaveStateMiddleware.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.Bot.Builder
     ///  Middleware that will call `read()` and `write()` in parallel on multiple `BotState`
```

Comment:
This comment, and the one on "OnTurnAsync" are misleading. While this method does have an LoadAsync method, that method needs to be manually called. Given that this is now "AutoSaveSet" we should remove that functionality, or document it more properly.